model-engine/model_engine_server/infra/gateways/datadog_monitoring_metrics_gateway.py

```diff
@@ -1,38 +0,0 @@
-from datadog import statsd
-from model_engine_server.core.config import infra_config
-from model_engine_server.domain.gateways.monitoring_metrics_gateway import MonitoringMetricsGateway
-
-
-class DatadogMonitoringMetricsGateway(MonitoringMetricsGateway):
```

Comment:
did this file get moved to internal?

Reply:
I suppose that dependencies on public vendors (e.g. Datadog) can be put in our public repo. It's the Scale-specific bits that need to be internal.

Reply:
yup, now a plugin